src/main/java/bio/terra/testrunner/runner/TestScriptResult.java

```diff
@@ -38,6 +39,24 @@ private TestScriptResultSummary(String testScriptName, String testScriptDescript
     }
   }
 
+  /** Store snapshots of UserJourneyResults, mainly used for reporting. */
+  @SuppressFBWarnings(
+      value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD",
+      justification = "This POJO class is used for easy serialization to JSON using Jackson.")
+  public static class TestScriptUserJourneySnapshots {
+    public String testScriptName;
+    public String testScriptDescription;
```

Comment:
Already there is a `TestScriptResult` class that has 2 properties: `List<UserJourneyResult>` and `TestScriptResultSummary`. The `TestScriptResultSummary` contains both of these properties. So isn't this duplicating the same information in a different object? I'm fine with modifying the output format, but would prefer not to duplicate information.